pdns/recursordist/docs/lua-scripting/dq.rst

```diff
@@ -193,6 +193,16 @@ The DNSQuestion object contains at least the following fields:
 
       Returns the :class:`DNSHeader` of the query or nil.
 
+  .. method:: DNSQuestion:getPolicyTags() -> {str}
```

Comment:
Now we have two entries describind `getPolicyTags`.... The order is indeed confusing. I think I re-ordered them in my routing tag PR.

Reply:
I'm pretty sure I messed up while rebasing, sorry!